e2e/.ignore

```diff
@@ -0,0 +1,8 @@
+# cargo-machete v0.7.0 detects unused dependencies in lcp.
+# Thus ignore linting for submodules until resolved the issue.
+# ---
+# Analyzing dependencies of crates in this directory...
+# cargo-machete found the following unused dependencies in this directory:
+# light-client -- ./e2e/lcp/modules/light-client/Cargo.toml:
+#         serde
+lcp/
```

Comment:
I didn't realize this. In cases where the `ibc` feature is not enabled, `serde` may not be necessary.